UnityProject/Assets/Scripts/Systems/Antagonists/Objectives/NotCuffed.cs

```diff
@@ -8,7 +8,11 @@ namespace Antagonists
 	[CreateAssetMenu(menuName="ScriptableObjects/AntagObjectives/NotCuffed")]
 	public class NotCuffed: Objective
 	{
-		protected override void Setup() {}
+		[SerializeField] private Objective escapeObject;
+		protected override void Setup()
+		{
+			Owner.AddObjectiveToAntag(escapeObject);
```

Comment:
surely this would only add If the person rolled  not cuffed?

Reply:
wdym?

Reply:
Objective setups are independent, it will only call the setup() function on the objective that exists for an antag.

Reply:
I don't get why you are adding to the antag objectives. It will already be in there is it is rolled.

Reply:
There can only be one escape objective, if the not cuffed objective is rolled im also adding the escape the shuttle objective with it

Reply:
what i don't understand is why you don't just make NotCuffed extend the escape objective?

Reply:
in my opinion they conceptually have nothing to do with each other

Reply:
Yeah don't need inheritance, but it makes sense that in order to "escape" you can't be in cuffs as you're not free.

Reply:
Completely forgot that that was a thing, done.